neural_modelling/src/neuron/builds/IF_cond_exp/Makefile

```diff
@@ -3,6 +3,8 @@ BUILD_DIR = build/
 NEURON_MODEL = $(SOURCE_DIR)/neuron/models/neuron_model_lif_impl.c
 NEURON_MODEL_H = $(SOURCE_DIR)/neuron/models/neuron_model_lif_impl.h
 INPUT_TYPE_H = $(SOURCE_DIR)/neuron/input_types/input_type_conductance.h
+NEURON_IMPL_H = $(SOURCE_DIR)/neuron/implementations/neuron_impl_standard.h
+SYNAPSE_IMPL_H = $(SOURCE_DIR)/neuron/implementations/synapse_impl_standard.h
 THRESHOLD_TYPE_H = $(SOURCE_DIR)/neuron/threshold_types/threshold_type_static.h
```

Comment:
synapse_impl.h / synapse_impl_standard.h no longer exist in the current version (after the add_input function was moved to the neuron implementation) so this and all subsequent references in further Makefiles - I'm not going to mark all of them! - can be removed.

Reply:
Fixed